samples/deploy/deployment/ingress-controller.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: ingress-controller
+      app: apisix
```

Comment:
the name `apisxia` is easy to conflict with the deployment of APISIX itself.
maybe you could change both `app` to `apisix-ingress-controller`  or something.

Reply:
@dayongi use `ingress-controller` is better